src/VirtualDevice.ts

```diff
@@ -52,6 +54,17 @@ export class VirtualDevice {
 
         return promise;
     }
+
+    public resetSession(): Promise<IVirtualDeviceResult> {
+        return this.message("alexa quit");
+    }
+
+    private normalizeTranscript(transcript: string): string {
+        if (!transcript) {
+            return "";
```

Comment:
I think this should return null, and we should update the spec to reflect that it can be null.

Legitimately, there are occasions where there is no audio reply to transcribe, and so null is the correct value to use.